pages/FilterAttribute/components/SingleSelect/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import View from 'Components/View';
+import Attribute from '../Attribute';
+import Item from '../Item';
+import ClearButton from '../ClearButton';
+import styles from '../../style';
+
+/**
+ * The Filter SingleSelect view component.
+ */
+class SingleSelect extends Component {
+  static propTypes = {
+    mergeTemporaryFilters: PropTypes.func.isRequired,
+    removeTemporaryFilter: PropTypes.func.isRequired,
+    temporaryFilters: PropTypes.shape().isRequired,
+    currentAttribute: PropTypes.shape(),
+  };
+
+  static defaultProps = {
+    currentAttribute: null,
+  };
+
+  /**
+   * Updates the current filter options selection.
+   * @param {Array} valueId A set of strings containing the current selected attribute options.
```

Comment:
i guess `valueId` is not an array here. please correct jsDoc

Reply:
Done